spec/index.html

```diff
@@ -2118,11 +2119,12 @@ <h5 id="the-coordinates-element">4.2.20 The <code>&lt;coordinates&gt;</code> ele
         </p>
         <p>
           The positions in the <code>coordinates</code> element define the location of the feature. For <code>geometry</code> elements whose value is , <code>linestring</code>, <code>multipoint</code> or <code>multilinestring</code>, this
-          specification neither assigns nor requires special ordering or other constraints, apart from the axis order described above. For <code>geometry></code> elements whose value is a <code>polygon</code> or
+          specification neither assigns nor requires special ordering or other constraints, apart from the axis order described above. For <code>geometry</code> elements whose value is a <code>polygon</code> or
           <code>multipolygon</code> element, the descendant <code>coordinates</code> elements must follow additional constraints.
         </p>
-        <h5 id="styling">4.3 Styling</h5>
+        <h2 id="styling">4.3 Styling</h2>
         <p>MapML documents can be styled with Cascading Style Sheets.</p>
+        <p class="note"><a href="https://github.com/Maps4HTML/MapML/issues/71">Issue 71: How does styling MapML with CSS work?</a></p>
```

Comment:
Added a note with a link to the discussion of how CSS should work with MapML for more exposure so people can discuss. If you don't want to include this here, let me know.

Reply:
Sounds reasonable, let's go with it.